tools/python/boutiques/bosh.py

```diff
@@ -0,0 +1,280 @@
+#!/usr/bin/env python
+
+from argparse import ArgumentParser, RawTextHelpFormatter
+import jsonschema
+import json
+import os, sys
+
+class BoutiquesTools():
```

Comment:
How about calling this `BoutiquesTool` (no plural)?

Reply:
actually, how about having a module instead of a class? Currently, we need to create an object to access all the functions below, but this object has no attribute.

Reply:
Conversation moved to your PR on this branch; https://github.com/gkiar/boutiques/pull/5